libs/sdk-react-native/ios/RNBreezSDK.swift

```diff
@@ -345,6 +345,16 @@ class RNBreezSDK: RCTEventEmitter {
             rejectErr(err: err, reject: reject)
         }
     }
+
+    @objc(lspInfo:resolver:rejecter:)
```

Comment:
Should mirror the RCT_EXTERN_METHOD, so:
```
    @objc(lspInfo:rejecter:)
```

Reply:
Fixed, thanks!